src/main/java/uk/gov/hmcts/cmc/claimstore/ClaimStoreApplication.java

```diff
@@ -7,7 +7,7 @@
 
 @SpringBootApplication(exclude = DataSourceAutoConfiguration.class)
 @SuppressWarnings({"HideUtilityClassConstructor", "squid:S1118"}) // Spring needs a constructor, its not a utility class
-@EnableFeignClients
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.cmc.claimstore", "uk.gov.hmcts.document"})
```

Comment:
I thought @dharmendrak said this wasn't needed anymore?

Reply:
sorry Tim, it is needed.